projects/1.18/assets/stoneworks/stoneworks/lang/zh_cn.json

```
@@ -0,0 +1,584 @@
+  "block.stoneworks.netherrack_bricks": "下界岩砖",
+  "block.stoneworks.polished_calcite": "磨制方解石",
+  "block.stoneworks.polished_calcite_slab": "磨制方解石台阶",
+  "block.stoneworks.polished_calcite_stairs": "磨制方解石楼梯",
+  "block.stoneworks.polished_calcite_wall": "磨制方解石墙",
+  "block.stoneworks.polished_dark_prismarine": "磨制暗海晶石",
+  "block.stoneworks.polished_dark_prismarine_slab": "磨制暗海晶石台阶",
+  "block.stoneworks.polished_dark_prismarine_stairs": "磨制暗海晶石楼梯",
+  "block.stoneworks.polished_dark_prismarine_wall": "磨制暗海晶石墙",
+  "block.stoneworks.polished_end_stone": "磨制末地石",
+  "block.stoneworks.polished_end_stone_slab": "磨制末地石台阶",
+  "block.stoneworks.polished_end_stone_stairs": "磨制末地石楼梯",
+  "block.stoneworks.polished_end_stone_wall": "磨制末地石墙",
+  "block.stoneworks.polished_netherrack": "磨制下界岩",
+  "block.stoneworks.polished_netherrack_slab": "磨制下界岩台阶",
+  "block.stoneworks.polished_netherrack_stairs": "磨制下界岩楼梯",
+  "block.stoneworks.polished_netherrack_wall": "磨制下界岩墙",
+  "block.stoneworks.polished_prismarine": "磨制海晶石",
+  "block.stoneworks.polished_prismarine_slab": "磨制海晶石台阶",
+  "block.stoneworks.polished_prismarine_stairs": "磨制海晶石楼梯",
+  "block.stoneworks.polished_prismarine_wall": "磨制海晶石墙",
+  "block.stoneworks.polished_purpur": "磨制紫珀块",
+  "block.stoneworks.polished_purpur_slab": "磨制紫珀块台阶",
+  "block.stoneworks.polished_purpur_stairs": "磨制紫珀块楼梯",
+  "block.stoneworks.polished_purpur_wall": "磨制紫珀块墙",
+  "block.stoneworks.polished_quartz": "磨制石英块",
+  "block.stoneworks.polished_quartz_slab": "磨制石英块台阶",
+  "block.stoneworks.polished_quartz_stairs": "磨制石英块楼梯",
+  "block.stoneworks.polished_quartz_wall": "磨制石英块墙",
+  "block.stoneworks.polished_tuff": "磨制凝灰岩",
+  "block.stoneworks.polished_tuff_slab": "磨制凝灰岩台阶",
+  "block.stoneworks.polished_tuff_stairs": "磨制凝灰岩墙",
+  "block.stoneworks.polished_tuff_wall": "磨制凝灰岩墙",
+  "block.stoneworks.prismarine_brick_slab": "海晶石砖台阶",
+  "block.stoneworks.prismarine_brick_stairs": "海晶石砖楼梯",
+  "block.stoneworks.prismarine_brick_wall": "海晶石砖墙",
+  "block.stoneworks.prismarine_bricks": "海晶石砖",
+  "block.stoneworks.prismarine_paver_slab": "海晶石地砖台阶",
+  "block.stoneworks.prismarine_paver_stairs": "海晶石地砖楼梯",
+  "block.stoneworks.prismarine_paver_wall": "海晶石地砖墙",
+  "block.stoneworks.prismarine_pavers": "海晶石地砖",
+  "block.stoneworks.prismarine_pillar": "海晶石柱",
+  "block.stoneworks.prismarine_plate_slab": "切制海晶石台阶",
+  "block.stoneworks.prismarine_plate_stairs": "切制海晶石楼梯",
+  "block.stoneworks.prismarine_plate_wall": "切制海晶石墙",
+  "block.stoneworks.prismarine_plates": "切制海晶石",
+  "block.stoneworks.prismarine_tile_slab": "台阶竹制海晶石台阶",
+  "block.stoneworks.prismarine_tile_stairs": "竹制海晶石楼梯",
+  "block.stoneworks.prismarine_tile_wall": "竹制海晶石墙",
+  "block.stoneworks.prismarine_tiles": "竹制海晶石",
+  "block.stoneworks.purpur_brick_slab": "紫珀块砖台阶",
+  "block.stoneworks.purpur_brick_stairs": "紫珀块砖楼梯",
+  "block.stoneworks.purpur_brick_wall": "紫珀块砖墙",
+  "block.stoneworks.purpur_bricks": "紫珀块砖",
```

```suggestion
  "block.stoneworks.purpur_brick_slab": "紫珀砖台阶",
  "block.stoneworks.purpur_brick_stairs": "紫珀砖楼梯",
  "block.stoneworks.purpur_brick_wall": "紫珀砖墙",
  "block.stoneworks.purpur_bricks": "紫珀砖",
```
不建议紫珀、石英、紫水晶等在后方加`块`
个人认为`块`本身指代`Block`，应当不是材料名的一部分
请全文修改

---

> 不建议紫珀、石英、紫水晶等在后方加`块` 个人认为`块`本身指代`Block`，应当不是材料名的一部分 如修改请全文修改

我把有后置定语的`块`移除了，但保留了没有后置定语的例如`磨制石英块`